src/core/lib/security/security_connector/local/local_security_connector.cc

```diff
@@ -124,19 +129,51 @@ static grpc_security_status local_auth_context_create(grpc_auth_context** ctx) {
 }
 
 static void local_check_peer(grpc_security_connector* sc, tsi_peer peer,
+                             grpc_endpoint* ep,
                              grpc_auth_context** auth_context,
                              grpc_closure* on_peer_checked) {
+  int fd = grpc_endpoint_get_fd(ep);
+  grpc_resolved_address resolved_addr;
```

Comment:
Is resolved_addr local address or peer address?

Reply:
It is a local address.

Reply:
Could you please verify that if local socket address is loopback, then the connection (i.e., peer) must be local.